src/mongocxx/test/CMakeLists.txt

```diff
@@ -196,6 +205,10 @@ add_test(transactions_specs test_transactions_specs)
 add_test(retryable_reads_spec test_retryable_reads_specs)
 add_test(read_write_concern_specs test_read_write_concern_specs)
 
+# Adding this as a test will run it as part of the RUN_TESTS command in MSVC.
+# Do not add, since we only test mongohouse on Linux.
```

Comment:
Why is this only on run on Linux? https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/atlas_data_lake accounts for running in Windows.

I am ok with limiting to Linux now. But I think it is worth testing connectivity on Windows / macOS since TLS goes through a different code path. But we could do this in C driver tests instead, since the C++ driver does not alter TLS / connection behavior.

Reply:
That... is a great question.  I was using Java's implementation as a reference, and they only run the mongohouse test on an ubuntu-1804.  But now that I've gone looking, I can't find anywhere that states which/how many different buildvariants should be running these tests.  

In the scope document for ADL, there are these two goals:
- For each driver, run the test suite in Evergreen triggered off of commits to ADL's master branch
- For each driver, run the test suite in Evergreen on every commit to the driver.

Nothing specific about what type of environment the test suite should run in on Evergreen.  We are satisfying the second goal, at least.  I'm not sure about how the first one would be triggered, but I didn't see any of the other implementations checking for updates to the ADL master branch.

For now, maybe we can leave this as just linux?  It would be easy to add additional environments later, once there's more clear guidance about which environments should be tested.

Reply:
> Nothing specific about what type of environment the test suite should run in on Evergreen. We are satisfying the second goal, at least. I'm not sure about how the first one would be triggered, but I didn't see any of the other implementations checking for updates to the ADL master branch.

Oh good catch! I think that is done outside of code changes, by setting an [inter-project dependency](https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#inter-project-dependencies) in [Evergreen project settings for cxx-driver](https://evergreen.mongodb.com/projects##cxx-driver). That can be done after this is merged.

For now, maybe we can leave this as just linux? It would be easy to add additional environments later, once there's more clear guidance about which environments should be tested.

Reply:
Will do!